programmers/난이도별/level02.전화번호_목록/Jaewon0702_전화번호 목록.py

```diff
@@ -0,0 +1,18 @@
+def solution(phone_book):
+    phone_book.sort()
+    for j,k in enumerate(phone_book):
+        for i,v in enumerate(phone_book):
+            if j!=i :
+                if k in v :
+                    return False
+                else :
+                    return True

```

Comment:
```suggestion
    phone_book.sort()
    for j,k in enumerate(phone_book):
        for i,v in enumerate(phone_book):
            if j != i :
                return not k in v;
```
boolean을 굳이 if/else로 구분해서 반환할 필요는 없어요!

Reply:
조금 더 보충하자면, j != i로 검사할 필요가 없어요! 어차피 똑같은 배열이기 때문이죠

```python
    phone_book.sort()
    for k in phone_book:
        for v in phone_book:
            if k != v : return not k in v;
```